src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -63,17 +65,18 @@
 
     private static final Logger LOGGER = Logger.getLogger(BaseAmazonWebServicesCredentials.class.getName());
 
-    public static final int STS_CREDENTIALS_DURATION_SECONDS = 3600;
+    public static final Integer STS_CREDENTIALS_DURATION_SECONDS = 3600;
```

Comment:
No need to change the type here, as it will be boxed when needed. Besides, I'm not sure if it would be backwards compatible. Better to revert this change.

Reply:
Ah, I changed type because I was getting boxing and unboxing compilation errors with the new `setStsTokenDuration` logic

Reply:
I’ll see what I can do to revert it when I get back to a computer

Reply:
Worst case: 
```
this.stsTokenDuration = stsTokenDuration == null || stsTokenDuration.intValue() == STS_CREDENTIALS_DURATION_SECONDS ? null : stsTokenDuration;
```